.github/CONTRIBUTING.md

```diff
@@ -138,7 +138,7 @@ Our pre-commit hooks verify that the linter and tests pass when committing.
 The `package.json` file contains various scripts for common tasks:
 
 - `yarn`: setup project by installing dependencies.
-- `yarn example-setup`: install dependencies for the example app
+- `yarn example setup`: install dependencies for the example app
```

Comment:
I moved this from being a command defined in the root project as `example-setup`, to a command in the example app's package.json as just `setup`. More appropriate placement. It still can be run from the root directory just without the dash (anything like `yarn example XYZ` just delegates XYZ to example)